source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -0,0 +1,306 @@
+.. _csharp-change-streams:
+
+====================
+Monitor Data Changes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: watch, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use a **change stream** to monitor real-time
+changes to your database. A change stream is a {+mdb-server+} feature that
```

Comment:
S: Since the next sentence clarifies that you can monitor collection, database, or deployment changes, I think it's confusing to just refer to database monitoring here. Maybe "data" would work better:
```suggestion
changes to your data. A change stream is a {+mdb-server+} feature that
```